src/compiler/bundle/typescript-plugin.ts

```diff
@@ -57,7 +57,10 @@ export const typescriptPlugin = (compilerCtx: d.CompilerCtx, bundleOpts: BundleO
           const tsResult = ts.transpileModule(mod.staticSourceFileText, {
             compilerOptions: config.tsCompilerOptions,
             fileName: mod.sourceFilePath,
-            transformers: { before: bundleOpts.customTransformers },
+            transformers: {
+              before: bundleOpts.customBeforeTransformers,
```

Comment:
should this `before` have a `?? []` too?

Reply:
https://github.com/ionic-team/stencil/pull/4501/commits/273f9caefb04da6757cc2e56ffd043bb2be0ab07